File: hacker-rank/3-months-preparation-kit/3-week/3-xor-strings-3/xor-strings-3.go
package main

import "fmt"

func main() {
	s1 := "10101"
	s2 := "00101"
	fmt.Println(strings_xor(s1, s2))
}

func strings_xor(s1 string, s2 string) string {
	var res string
	for i := 0; i < len(s1); i++ {
		if s1[i] != s2[i] {
			res += "1"
		} else {
			res += "0"
		}
	}
	return res
}

File: hacker-rank/3-months-preparation-kit/3-week/3-xor-strings-3/xor-strings-3.py
def strings_xor(s, t):
    res = ""
    for i in range(len(s)):
        if s[i] == t[i]:
            res += '0';
        else:
            res += '1';

    return res

s = input()
t = input()
print(strings_xor(s, t))

# El ejercicio en Golang no funciona pero trata de debuggear el metodo el cual hice los siguientes cambios
# Cambio 1:
#  if s[i] = t[i]:
# por 
#  if s[i] == t[i]: 
#Cambio 2:
#res = '0';
#por
#res += '0';
#Cambio 3:
#res = '1';
#por
#res += '1';

